gapis/trace/tracer/tracer.go

```diff
@@ -84,9 +84,15 @@ func LayersFromOptions(ctx context.Context, o *service.TraceOptions) []string {
 	if o.PerfettoConfig == nil {
 		return ret
 	}
+
+	added_layer := make(map[string]bool)
 	for _, x := range o.PerfettoConfig.GetDataSources() {
 		if layer, err := layout.LayerFromDataSource(x.GetConfig().GetName()); err == nil {
 			if _, err := layout.LibraryFromLayerName(layer); err == nil {
+				if added_layer[layer] {
+					continue
+				}
+				added_layer[layer] = true
```

Comment:
This is OK, but the more, sigh, idiomatic go way to do this is:
```
added := map[string]struct{}{}
for _, x := ... {
  ...
  if _, ok := added[layer]; !ok {
    ret = append(ret, layer)
    added[layer] = struct{}{}
  }
}
```
(also, use camelCase not snake_case)

Reply:
Thanks! Done.